packaging/after-install.sh

```diff
@@ -4,7 +4,10 @@ getent group nagios > /dev/null || groupadd --system nagios
 useradd --system ncr --home /opt/ncr --groups nagios
 
 if [ -x /bin/systemctl ]; then
+	/bin/systemctl daemon-reload
 	/bin/systemctl start ncr.service
+	/bin/systemctl enable ncr.service
```

Comment:
Yeah, that'll do it. I agree it should probably be doing that, considering it's also starting it on installation. Enabling services is pretty in-line with Debian policies.